Tasks/AzureResourceGroupDeploymentV2/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 141,
-        "Patch": 3
+        "Patch": 4
```

Comment:
Just realized one thing and it will impact previous PR as well.
We need to choose a patch version which is not already published otherwise the version will get ignored by tfs service when loading task

Reply:
One way to check would be to look in codex-depa feed and see if this version package  is already available, if yes then increment patch and iterate till you get the right one

Reply:
updated to 5. https://mseng.visualstudio.com/AzureDevOps/_packaging?_a=package&feed=Codex-Deps&package=Mseng.MS.TF.DistributedTask.Tasks.AzureResourceGroupDeploymentV2&protocolType=NuGet&version=2.182.0&view=versions